SU-HSDO/suhumsci
# shs-140: fixes photo album caption bug

## Summary
This fixes a bug where captions in a photo album wouldn't display if the user was logged out.

## Steps to Test
1. Go to `/components/photo-album` **not logged in - as anonymous user -**. 
2. Scroll through the slider and verify you can see captions. Some slides don't have captions, but if you can see any, it fixes the bug.

## Screenshots
![Screen Shot 2023-02-15 at 1 40 08 PM](https://user-images.githubusercontent.com/8405274/219122518-60cb3a16-43b5-4c34-976e-8b995c53ef8a.png)
